src/i18n/de.json

```diff
@@ -101,13 +101,13 @@
   "org": {
     "add": "Organisation hinzufügen",
     "addNode": "Sensor registrieren",
-    "addSite": "Standort hinzufügen",
-    "all": "All Ihre Organisationen",
+    "addSite": "Ort hinzufügen",
```

Comment:
Why change it back to Ort? From the Feedback we got it should be "Standort".

Reply:
ok, then we need to use "Standort" consistently. I'll introduce it everywhere.